sdk/src/main/java/com/kin/ecosystem/base/KinEcosystemInitiator.java

```diff
@@ -19,7 +21,11 @@ protected static void init(Context context) {
 		try {
 			Kin.initialize(context);
 			// If we had process restart we should load the account.
-			BlockchainSourceImpl.getInstance().createAccount();
+			KinAccount account = BlockchainSourceImpl.getInstance().getKinAccount();
+			if (account == null) {
+				final String kinUserId = AuthRepository.getInstance().getEcosystemUserID();
+				BlockchainSourceImpl.getInstance().loadAccount(kinUserId);
```

Comment:
why do we have 2 points that calls BlockchainSourceImpl.getInstance().loadAccount?